packages/ui/src/components/va-button/VaButton.stories.ts

```diff
@@ -1,12 +1,367 @@
-import { defineComponent } from 'vue'
-import VaButton from './VaButton.demo.vue'
+import { VaButton } from './'
```

Comment:
I added some changes with no-verify, but please make sure to fix tests before merge.